### ISSUE TITLE
CloudFile: localization problem

### ISSUE THREAD

**Issue by dartraiden** [miranda-ng/miranda-ng#805]
![2017-04-26_04h20_22](https://cloud.githubusercontent.com/assets/5046855/25414054/c9ebd796-2a37-11e7-9fff-d8f792b66b24.png)

This one should be english and translatable.  Same in Scriver:
[Yandex]
Яндекс

**Comment by dartraiden** [comment]
Strange, but it still not in langpack

Seems like lpgen.js filter this?

```
//filter _T() function results
function filter_T(string) {
//filter for exact matched strings
var filter1=/^(&?[Oo][Kk]|AOL|APOP|BBS|Bing|CTCP|DCC|Foodnetwork|Google|GPG|Hotmail|ICQ|ICQ Corp|ID|IP|ISDN|iTunes|Jabber|JID|Miranda|MirandaG15|Miranda NG|mRadio|MSN|NickServ|OSD|OTR|PCS|PGP|SMS|SSL|SteamID|Steam Guard|Tlen|Tox|Twitter|Yahoo|WhatsApp|Winamp \(\*\)|Windows|X400)$/g;
//filter string starting from following words
var filter2=/^(SOFTWARE\\|SYSTEM\\|http|ftp|UTF-|utf-|TEXT|EXE|exe|txt|css|html|dat[^a]|txt|MS\x20|CLVM|TM_|CLCB|CLSID|CLUI|HKEY_|MButton|BUTTON|WindowClass|MHeader|RichEdit|RICHEDIT|STATIC|EDIT|CList|listbox|LISTBOX|combobox|COMBOBOX|TitleB|std\w|iso-|windows-|<div|<html|<img|<span|<hr|<a\x20|<table|<td|miranda_|kernel32|user32|muc|pubsub|shlwapi|Tahoma|NBRichEdit|CreatePopup|&?[Oo][Kk]|<\/|<\w>|\w\\\w|urn\:|<\?xml|<\!|h\d|\.!\.).*$/g;
//filter string ending with following words
var filter3=/^.+(001|\/value|\*!\*|=)$/g;
//filter from Kildor
var filter4=/^((d\s\w)|\[\/?(\w|url|img|size|quote|color)(=\w*)?\]?|(\\\w)|(%\w+%)|(([\w-]+\.)*\.(\w{2,4}|travel|museum|xn--\w+))|\W|\s|\d)+$/gi;
//filter from Kildor for remove filenames and paths.
var filter5=/^[\w_:%.\\\/*-]+\.\w+$/g;
```

**Comment by dartraiden** [comment]
Fixed in 72ff17b7779541d67465eb0495756e3bc13b6082
